docs/releasing/publishing-a-pre-release.md

```diff
@@ -0,0 +1,135 @@
+# Before you publish a pre-release of GOV.UK Frontend
+
+Decide the kind of pre-release you're publishing. All pre-releases are public, but we distinguish:
+
+- `internal` pre-releases for internal use
+- `beta` pre-releases aimed at external users
+
+We'll use the kind of pre-release as [the pre-release identifier](https://docs.npmjs.com/cli/v8/commands/npm-version#preid) in the package version.
+
+## If you're publishing a beta pre-release
+
+Besides publishing the code itself, beta releases will likely involve
```

Comment:
Can we pin this down so we can say something like:

```md
Much like an actual release, a beta release involves:

- publishing the code itself
- documentation updates linked to the code
- user communication
```

If we're not sure about pinning it down, the `similarly to an actual release` is a bit awkward. Maybe something like:

```
Besides publishing the code itself, beta releases will likely involve code-related documentation updates and user communications, much like an actual release.